lib/approvals/verifiers/json_verifier.rb

```diff
@@ -0,0 +1,27 @@
+module Approvals
+  module Verifiers
+    class JsonVerifier
+      def initialize(received_path, approved_path)
+        self.received_path = received_path
+        self.approved_path = approved_path
+      end
+
+      def verify
+        return false unless approved == received
+        true
+      end
```

Comment:
I'm curious why this couldn't be

```diff
       def verify
-        return false unless approved == received
-        true
+       approved == received
       end
```

Reply:
it definitely can... no idea why i wrote it that way. will change it